src/sentry/static/sentry/app/components/actions/ignore.jsx

```diff
@@ -173,7 +173,7 @@ export default class IgnoreActions extends React.Component {
                   return (
                     <li className="dropdown-submenu" key={count}>
                       <DropdownLink
-                        title={t('%s times', count.toLocaleString())}
+                        title={tn('one time', '%s times', count)}
```

Comment:
Should this be consistent with the other menu items?

```suggestion
                        title={tn('1 time', '%s times', count)}
```

Reply:
My personal preference is spelling out one (I think theres some English best practice there), but I'm not opposed.

Anyone else have thoughts?

Reply:
I prefer `1 time` for consistency